src/components/DisplayContainer/DisplayContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './DisplayContainer.scss'
+import MovieCard from '../MovieCard/MovieCard';
+
+
+const DisplayContainer = ( {movies} ) => {
+  const movieCards = movies.map(movie => {
+    return <MovieCard
+      key={movie.episode_id}
+      episode={movie.episode_id}
+      title={movie.title}
+      releaseDate={movie.release_date}
+      />
+  })
+  
+  return (
+    <main className='display-container' key=''>
```

Comment:
We need to pass a key as a prop on line 9 to the MovieCards (looks great how you set this up!) but on line 17 - do we need the `key` property?

Reply:
As we implement more functionality/conditional rendering, will we need to use state for `currentDisplayedPage` and `selectedMovie`?  I don't have a clear preferred path for this yet but let's brainstorm together on how/where to do this.  

-Could give each MovieCard state of `selected: T/F`
-Could give App or Display Container state of `selectedMovie: '4' using the episode number
-Will likely need to give App state of `currentDisplayedPage` to render accordingly from there